conf/test.config

```diff
@@ -10,20 +10,30 @@
 ----------------------------------------------------------------------------------------
 */
 
+process {
+    withName: 'PURPLE' {
+        ext.args = '-min_purity 1 -max_purity 1 -min_ploidy 2 -max_ploidy 2'
+    }
+}
+
 params {
     config_profile_name        = 'Test profile'
     config_profile_description = 'Minimal test dataset to check pipeline function'
 
+    // NOTE(SW): incompatible with GHA given size of reference data, STAR align requires ~30 GB memory
     // Limit resources so that this can run on GitHub Actions
-    max_cpus   = 2
-    max_memory = '6.GB'
+    max_cpus   = 1
+    max_memory = '30.GB'
```

Comment:
I've been unable to run an end-to-end test for the CI since the required reference is too large for the GHA runners. And you're right that the 30.GB request is also too high for the default runners. Instead, this is something I run locally for testing.

The current test executed by the GHA workflow is only a stub test. This is something I planned to improve after the initial release - I suspect it will take a significant amount of time to reduce reference data size so that end-to-end tests work on the restrictive default runners

Reply:
Adding backlink to the corresponding issue:

- https://github.com/nf-core/oncoanalyser/issues/71